src/pages/Admin/Room/Edit/RoomEditModal.tsx

```diff
@@ -183,6 +184,25 @@ export const RoomEditModal: React.FC<RoomEditModalProps> = ({
             </div>
           </Form.Row>
 
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Change room type</Form.Label>
+              <Form.Control
+                as="select"
+                name="type"
+                ref={register}
+                defaultValue={room.type}
```

Comment:
Will this correctly select the default value when `room.type === undefined`? Or should we have `?? ""` here?

Reply:
Yes, if value is null or undefined, the default option is selected.

Reply:
Cool, makes sense.